examples/some-verifier/src/db.rs

```diff
@@ -0,0 +1,121 @@
+use some_verifier_lib::Platform;
+use tokio_postgres::{NoTls, Row};
+
+/// A social media platform that can be stored in the database.
+pub trait DbPlatform {
+    /// The `Platform` enum value of the platform.
+    const PLATFORM: Platform;
+}
+
+/// A trait that is implemented for the Platform enum to give some utility functons.
+trait DbName {
+    /// The name of the corresponding table.
+    fn table_name(&self) -> &'static str;
+    /// The name of the corresponding column.
+    fn column_name(&self) -> &'static str;
+}
```

Comment:
I don't get why you don't just implement these as methods on the `Platform` type.

THis is an awful lot of complexity and needless use of macros.

Reply:
Will be fixed, when #17 is eventually merged.